syntax = "proto3";

package resdb.storage;

message RocksDBInfo {
  uint32 num_threads = 2;
  uint32 write_buffer_size_mb = 3;
  uint32 write_batch_size = 4;
  string path = 5;
}

